core/embed/rust/src/ui/model_mercury/component/button.rs

```diff
@@ -31,6 +31,7 @@ pub struct Button {
     content: ButtonContent,
     styles: ButtonStyleSheet,
     text_align: Alignment,
+    radius: Option<i16>,
```

Comment:
Changed the type. 
Regarding the radius: we removed it `ButtonStyleSheet` since we thought all mercury buttons will be rectangles. However, radius does not change for `normal` / `active` / `disabled` states, and also it's only used for the passphrase "Confirm" button so I don't know. Maybe it would be better to render it in `passphrase.rs` as is done in `bip39.rs` and `slip39.rs` where the "pill-shaped" button changes size with changing suggestions.

Reply:
Fixed on rebase.